authentication/serializers.py

```
@@ -0,0 +1,128 @@
+                password=password,
+                first_name=validated_data.get('first_name', ''),
+                last_name=validated_data.get('last_name', ''),
+                is_active=True,
+                is_verified=False,
+                is_registered=False
+            )
+            
+        except Exception as e:
+            raise serializers.ValidationError({
+                'detail': f'Failed to create user: {str(e)}'
+            })
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)